frontend/src/components/HandbookOverlay/HandbookPage.css

```diff
@@ -34,3 +34,10 @@
 	background: var(--handbook-locked-section-colour);
 	text-align: center;
 }
+
+.system-role-error-message {
```

Comment:
To be safe, I'd prefer this selector to include the handbook-terms class:

```
.handbook-terms .system-role-error-message {
```

Mind you, we already have inconsistency here because we should be doing that for role-locked as well. That's an important one because it's generic enough that it could leak style rules... So could you fix that one here as well please? Thanks!

```
.handbook-terms .term .role-locked {
```